test/sync/src/commonMain/kotlin/io/realm/kotlin/test/mongodb/util/AdminApi.kt

```diff
@@ -94,14 +107,9 @@ interface AdminApi {
     suspend fun changeSyncPermissions(permissions: SyncPermissions, block: () -> Unit)
 
     /**
-     * Set whether or not automatic confirmation is enabled.
-     */
-    suspend fun setAutomaticConfirmation(enabled: Boolean)
-
-    /**
-     * Set whether or not custom confirmation is enabled.
+     * Set type of authentication method used for the EmailAuth provider.
      */
-    suspend fun setCustomConfirmation(enabled: Boolean)
+    suspend fun setAuth(auth: UserConfirmationMode)
```

Comment:
Do we even use this anymore?

Reply:
No, I was a bit afraid of removing it, but I should just get rid of it.